docs/Data_Submission_Portal/Users_Guide/Troubleshooting.md

```diff
@@ -0,0 +1,1236 @@
+                        "message":"Key 'consent_typ' is not a valid property for type 'case'. Did you mean 'consent_type'?",
+                        "type":"INVALID_PROPERTY"
+                    },
+                    {
+                        "keys":[],
+                        "message":"Additional properties are not allowed ('consent_typ' was unexpected)",
+                        "type":"ERROR"
+                    }
+                ],
+            "id":
+                [
+                    {
+                        "id":"a00f076e-d694-47dd-8e50-24c28e90fd6a"
+                    }
+                ],
+            "related_cases":[],
+            "type":"case",
+            "unique_keys":
+                [
+                    {
+                        "project_id":"GDC-INTERNAL",
+                        "submitter_id":"GDC-INTERNAL-000093"
+                    }
+                ],
+            "valid":false,
+            "warnings":[]
+        }
+    ],
+    "entity_error_count":1,
+    "message":"Transaction aborted due to 1 invalid entity.",
+    "success":false,
+    "transaction_id":6372996,
+    "transactional_error_count":0,
+    "transactional_errors":[],
+    "updated_entity_count":0
+}
+```
+```Request2
+{
+  "projects": {
+    "code": "INTERNAL"
+  }, 
+  "disease_type": "Adenomas and Adenocarcinomas", 
+  "primary_site": "Bladder",
+  "type": "case",
+  "project_id": "GDC-INTERNAL", 
+  "submitter_id": "GDC-INTERNAL-000093",
+  "consent_type": "Informed Consent"
+}
+```
+```Response1
+{
+  "cases_related_to_created_entities_count": 0,
+  "cases_related_to_updated_entities_count": 0,
+  "code": 200,
+  "created_entity_count": 0,
+  "entities":
+    [
+        {
+            "action":"update",
+            "errors":[],
+            "id":
+                [
+                    {
+                        "id":"a00f076e-d694-47dd-8e50-24c28e90fd6a"
+                    }
+                ],
+            "related_cases":[],
+            "type":"case",
+            "unique_keys":
+                [
+                    {
+                        "project_id":"GDC-INTERNAL",
+                        "submitter_id":"GDC-INTERNAL-000093"
+                    }
+                ],
+            "valid":true,
+            "warnings":[]
+        }
+    ],
+    "entity_error_count":0,
+    "message":"Transaction successful.",
+    "success":true,
+    "transaction_id":6372997,
+    "transactional_error_count":0,
+    "transactional_errors":[],
+    "updated_entity_count":1
+}
+```
+
+#### INVALID_TYPE Messages
+
+|Message|Explanation|Solution|
+| --- | --- | --- |
+| __missing 'type'__ | This is a vague error message that frequently does not encapsulate the reason that the data upload is failing | Scrutinize the file for other issues such as a formatting problem (e.g. an extra column in the TSV, incorrect JSON formatting, the node is already in state=submitted, or a case is not registered with dbGaP) |
```

Comment:
I would recommend adding to this sentence:  This is a vague error message that frequently does not encapsulate the reason that the data upload is failing. +  This may be indicative of multiple errors occurring simultaneously.